### PR TITLE
style(theme/Tag): set background style on close button hover

### LINKED ISSUE #10266: [Tag] Update close button styles
Update the styles for the close button inside the Tags.

image

https://github.com/ethereum/ethereum-org-website/assets/1120748/0474f8e4-8127-46a0-8983-b2239d259ab4

DS: https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System?type=design&node-id=1600-34738&t=thqLEloXh2D9Dd4H-0

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Updates the close button style on hover to sync with the DS.

This is not a complete sync — see the "Normal" and "Tag" Solid variants. This provided additional complication that already exists in the theming.

## Related Issue

- Closes #10266

### CODE REVIEW

**Comment by TylerAPfledderer** [comment]
@pettinarip and @nloureiro, I would like to try and review the tag styling with you, and present an alternative that I have made (Pablo has seen this).

What makes creating the styling in the code with all the variants is inconsistency in how all the different colors are being used. The generates a bloated set of styles that have to be hard-coded. I would like to present a solution that reduces the colors used to just the two different shades in the palette for each "status" type which would also improve a11y in the color contrast, especially with this hover styles of the close button.

File: yarn.lock
Comment (pettinarip) [inline yarn.lock:1]: we shouldn't have this file modified

**Review by pettinarip — CHANGES_REQUESTED** [review]
@TylerAPfledderer nice, lgtm. Once we revert the `yarn.lock` we could merge this.


=====
Not related to this PR but noted a bug on the Tags as links
image

When hovered, it is displaying the text with the Link styles. Not a big deal because we are not currently using this case on the site.

**Comment by pettinarip** [comment]
> @pettinarip and @nloureiro, I would like to try and review the tag styling with you, and present an alternative that I have made (Pablo has seen this).
> 
> What makes creating the styling in the code with all the variants is inconsistency in how all the different colors are being used. The generates a bloated set of styles that have to be hard-coded. I would like to present a solution that reduces the colors used to just the two different shades in the palette for each "status" type which would also improve a11y in the color contrast, especially with this hover styles of the close button.

Sounds good to me!

FYI new colors are coming to the DS. Just saying for context but I don't think it will affect what you're suggesting here.

File: yarn.lock
Comment (TylerAPfledderer) [reply yarn.lock:1]: @pettinarip The file modified when I brought in the latest changes from `dev` and ran `yarn install`

Comment (TylerAPfledderer) [reply yarn.lock:1]: @pettinarip merging the latest `dev` still provided a diff for the lockfile

**Review by pettinarip — APPROVED** [review]
Fixed. The difference was added in your first commit, I added a revert for that file.

The rest looks good, approving 👍🏼

**Pull request merged**